packages/amplify-console-hosting/event-handlers/handle-PostInit.js

```diff
@@ -0,0 +1,10 @@
+const eventName = 'PostInit';
```

Comment:
Do you need this file?

Reply:
No I don't use any of them. Can be deleted